chef_master/source/perform_compliance_scan.rst

```diff
@@ -131,46 +69,91 @@ Before you can see if your nodes are compliant, you need to have the profiles yo
-           "compliance": "<username>/linux-baseline"
-         },
-         {
-           "name": "DevSec SSH Baseline",
-           "compliance": "<username>/ssh-baseline"
-         }
-       ],
-       "reporter": "chef-server-automate"
-     }
-   }
+  chef generate cookbook mycompany_audit
 
-.. note:: The ``audit`` cookbook also requires a time synchronization policy to be in place, such as Network Time Protocol (NTP).
+Place the following in mycompany_audit/metadata.rb:
 
-You can add the ``audit::default`` recipe to an existing run-list; however, in the example below, we will bootstrap a node with the ``audit::default`` recipe and the JSON shown above to do a series of baseline checks against a new node.
+.. code-block:: ruby
+
+  depends 'audit'
+
+Add the following in mycompany_audit/recipes/default.rb:
+
+.. code-block:: ruby
+
+  include_recipe 'audit::default'
+
+Generate the default attributes file:
+
+.. code-block:: ruby
+
+  chef generate attribute default
+
+Configure the ``audit`` cookbook reporter to send scan data to Automate in the mycompany_wrapper/attributes/default.rb file. ``audit`` cookbook supports many different configurations, see :doc:`Supported Configurations </audit_supported_configurations>`.
+
+.. code-block:: ruby
+
+  default['audit']['reporter'] = 'chef-server-automate'
+
+To run the baseline compliance profile for Windows or Linux, depending on which OS runs on the node,
+add the following code in mycompany_audit/attributes/default.rb:
+
+.. code-block:: ruby
+
+  default['audit']['reporter'] = 'chef-server-automate'
+  case node['os']
+  when 'linux'
+    default['audit']['profiles'] = [
+     {
+        'name': 'DevSec Linux Security Baseline',
+        'compliance': 'admin/linux-baseline'
+      }
+    ]
+    when 'windows'
+    default['audit']['profiles'] = [
+      {
+        'name': 'DevSec Windows Security Baseline',
+        'compliance': 'admin/windows-baseline'
+      }
+    ]
+  end
+
+
+Use Berkshelf to install cookbook dependencies and upload it to all Chef Servers.
+
+.. code-block:: ruby
+
+  berks install && berks upload
+
+Collect Compliance Scan Data
++++++++++++++++++++++++++++++++++++++++
+You can add the ``mycompany_wrapper::default`` recipe to an existing run-list; however, in the example below, we will bootstrap a node with the ``mycompany_wrapper::default`` recipe and run a series of baseline checks against a new node.
 
 .. code-block:: bash
 
-   $ knife bootstrap localhost --ssh-port 2201 --ssh-user vagrant --sudo --identity-file /Users/<username>/Repos/chef-test/.vagrant/machines/node1-ubuntu/virtualbox/private_key --node-name node1-ubuntu --json-attribute-file files/test_audit.json --run-list 'recipe[audit::default]'
+   knife bootstrap localhost --ssh-port 2201 --ssh-user vagrant --sudo --identity-file /Users/<username>/Repos/chef-test/.vagrant/machines/node1-ubuntu/virtualbox/private_key --node-name node1-ubuntu --run-list 'recipe[mycompany_wrapper::default]'
```

Comment:
No need for such a complicated command. It should be:

```
knife bootstrap NODE_IP_OR_FQDN --ssh-user ADMIN_USERNAME --sudo --identity-file ~/.ssh/private_key.pem --run-list 'recipe[mycompany_wrapper::default]'
```

or:

```
knife bootstrap NODE_IP_OR_FQDN -x ADMIN_USERNAME -P ADMIN_PASSWORD --sudo -r 'recipe[mycompany_wrapper::default]'
```

Also, there are quite a few ways to bootstrap nodes. We should link: https://docs.chef.io/knife_bootstrap.html

Reply:
ok.